packages/comps-db/src/schema.ts

```diff
@@ -16,6 +16,16 @@ import {
   varchar,
 } from "drizzle-orm/pg-core";
 
+// Add the enum type for cross chain trading
+export const CrossChainTradingType = {
+  DISALLOWALL: "DISALLOWALL",
+  DISALLOWXPARENT: "DISALLOWXPARENT",
+  ALLOW: "ALLOW",
+} as const;
+
+export type CrossChainTradingType =
+  (typeof CrossChainTradingType)[keyof typeof CrossChainTradingType];
+
```

Comment:
Ok so I think the shape of the changes in other files of this PR looks good. But this isn't the way I would implement this change at the db level. 

I'd recommend using Drizzle's `pgEnum` which creates a Postgres enum type under the hood and results in type safe data from the database onward.

Reply:
```suggestion
export const crossChainTradingType = pgEnum("cross_chain_trading_type", [
  "disallowAll",
  "disallowXParent",
  "allow",
]);

export type CrossChainTradingType =
  (typeof crossChainTradingType.enumValues)[number];
```

Reply:
Nice - thank you!